spyder/widgets/mixins.py

```diff
@@ -496,14 +514,17 @@ def show_tooltip(self, title=None, signature=None, text=None,
             inspect_word=inspect_word,
             display_link=display_link,
             max_lines=max_lines,
+            width=width
         )
 
         self._update_stylesheet(self.tooltip_widget)
 
         # Display tooltip
         self.tooltip_widget.show_tip(point, tiptext, cursor=cursor)
 
-    def show_hint(self, text, inspect_word, at_point):
+    def show_hint(self, text, inspect_word, at_point,
+                  max_lines=30,
+                  width=120):

```

Comment:
Any particular reason for doing

```python
    def show_hint(self, text, inspect_word, at_point,
                  max_lines=30,
                  width=120)
```

as opposed to 

```
    def show_hint(self, text, inspect_word, at_point,
                  max_lines=30, width=120)
```
?

Reply:
I was testing what @ccordoba12  commented [here](https://github.com/spyder-ide/spyder/pull/9585#discussion_r294055120) and before that I probably have there the constants `_DEFAULT_MAX_LINES` and `_DEFAULT_WIDTH` that take more space than the numbers (and I only replaced the values without reformating the params into only one line).